src/test/java/Tests/object/BCommentTable.java

```diff
@@ -62,6 +65,18 @@ public boolean isCommentInactive(String state) {
         }
     }
 
-}
-
+    @Step("Verify presence of the category in comment categories ")
+    public boolean verifyCategories(String expected) throws NoSuchElementException {
+        try {
+            List<WebElement> categories = this.driver.findElements(By.xpath("//*[@class='categorycolumn']"));
+            for (WebElement row : categories) {
+                System.out.println(this.driver.findElement((By.xpath(String.format("//*[@class='categorycolumn' and contains(text(), \"%s\")]", expected)))));
```

Comment:
If this line returns false, do you still need to continue? 

Please remove all `System.out.println`.

Reply:
fixed